crates/proof-of-sql/examples/posql_db/main.rs

```diff
@@ -149,14 +152,19 @@ fn main() {
     println!("Warming up GPU...");
     blitzar::compute::init_backend();
     println!("Done.");
+
+    let mut rng = thread_rng();
+    let public_parameters = PublicParameters::rand(5, &mut rng);
+    let prover_setup = ProverSetup::from(&public_parameters);
+    let verifier_setup = VerifierSetup::from(&public_parameters);
```

Comment:
This is the reason that the verification is failing. Because you are using `thread_rng`, the public parameters are different every time this is run. As a result the prover setup and the verifier setup have a mismatch.

To fix this, we just need `rng` to be a deterministic (seeded) random number generator. That should fix this.

Reply:
Updated the code to use the `rng`. Now the errors are resolved and we can verify the proofs